src/container/mq-topNavigation.jsx

```diff
@@ -0,0 +1,30 @@
+import { faBars, faUser } from "@fortawesome/free-solid-svg-icons";
+import React, { useContext, useState } from "react";
+import { InnerContainerRow, Navigation, NavigationContainer, NavigationElement, NavigationSpacer } from "../components/bottomnavigation";
+import { useHistory } from "react-router-dom";
+import { AuthenticationContext } from "../components/authentication";
+import { Sidebar } from "./sidebar";
+
+export const MQContext = React.createContext({})
+export const MQTopNavigation = () => {
```

Comment:
MQTopNavigation doesn't really explain what it is doing, just that it is a top navigation element? Is the MQ prefix necessary?